sw/host/opentitanlib/src/transport/verilator/subprocess.rs

```diff
@@ -94,3 +94,35 @@ impl Subprocess {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    fn echo_subprocess() -> Result<Subprocess> {
+        let options = Options {
+            executable: "/bin/echo".to_owned(),
+            rom_image: "".to_owned(),
+            flash_image: "".to_owned(),
+            otp_image: "".to_owned(),
+            extra_args: vec!["abc 123 def 456".to_owned()],
+        };
+        Subprocess::from_options(options)
+    }
+
+    #[test]
+    fn test_find_regex() -> Result<()> {
+        let mut subprocess = echo_subprocess()?;
+        let regex = Regex::new("abc (.*) def")?;
+        let found = subprocess.find(&regex, Duration::from_millis(100))?;
```

Comment:
Can we make the timeout significantly larger since we don't really care about the timing? 100ms should be fine but heavily loaded VMs, such as those used in CI systems, can be unpredictable.

Reply:
Set to 5000 ms.